bindings/nodejs/tests/secret_manager/secret_manager.spec.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import 'reflect-metadata';
+
+import { describe, it, expect } from '@jest/globals';
+import {
+    CoinType,
+    SecretManager,
+    Utils,
+} from '../../lib/';
+
+describe('SecretManager', () => {
+    it('generate IOTA Ed25519 address', async () => {
+        const mnemonicSecretManager = {
+            mnemonic: "acoustic trophy damage hint search taste love bicycle foster cradle brown govern endless depend situate athlete pudding blame question genius transfer van random vast"
+        };
+
+        let bech32_hrp = Utils.iotaMainnetProtocolParameters().bech32Hrp;
+
+        const secretManager = SecretManager.create(mnemonicSecretManager);
+        const addresses = await secretManager.generateEd25519Addresses({
```

Comment:
Use singular version?

Reply:
there's no singular version anymore

Reply:
uhhh why?

Reply:
... but that should be the follow-up PR. Do you think I should bring it back here?

Reply:
It's fine for a follow up